docs/compile.md

```diff
@@ -15,15 +15,15 @@ If the contract has a lot of dependencies it can take a while to compile - so yo
 
 ![](images/a-sol-compiler.png)
 
-After each compilation, a list is updated with all the newly compiled
-contracts.  The contract compiled can be selected with the Contract pulldown menu ( **F. in image below**).  Multiple contracts are compiled when one contract imports other contracts.  Selecting a contract will show information about that one.
+After each compilation, a list is updated with all newly compiled
+contracts.  The compiled contract can be selected with the Contract pulldown menu ( **F. in image below**).  Multiple contracts are compiled when one contract imports other contracts.  Selecting a contract will show information about that one.
```

Comment:
I think we need a further edit:
A compiled contract can be selected... instead of The compiled contract